js-async-functions/js-async-functions_sleep/js/index.js

```diff
@@ -8,19 +8,27 @@ const body = document.body;
 
 button3.addEventListener("click", async () => {
   icon.textContent = "💤";
-  sleep(3);
+  body.classList.add("dark");
+  await sleep(3);
   icon.textContent = "⏰";
+  body.classList.remove("dark");
 });
 
-button5.addEventListener("click", () => {
+button5.addEventListener("click", async () => {
   icon.textContent = "💤";
+  body.classList.add("dark");
+  await sleep(5);
   icon.textContent = "⏰";
+  body.classList.remove("dark");
 });
 
-function handleSleepFor10Button() {
+const handleSleepFor10Button = async () => {
```

Comment:
hätte schon gereicht wenn Du einfach "async" for die function schreibst und dann den Inhalt der Funktion genau so aufbaust wie bei den button Event Listenern davor - aber sofern es funktioniert all good :)